modalstyle.css

```diff
@@ -0,0 +1,195 @@
+@media (max-width: 768px) {
+  .popup {
+    z-index: 100;
+    position: fixed;
+    background-color: #fff;
+    width: 100vw;
+    height: 100vh;
+    overflow-y: auto;
+    top: 0;
+    left: 0;
+  }
+
+  .modalbox {
+    display: flexbox;
+    position: relative;
+    align-items: center;
+    padding: 3%;
+    margin: 5% auto;
+  }
+
+  .modal-links {
+    display: inline-block;
+    gap: 10px;
+    margin-bottom: 20px;
+  }
```

Comment:
- [ ] Kindly follow the given [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9), to `style the fonts, the position of the text and for margin/padding`.
***Figma Design***
![Screenshot from 2023-01-06 04-57-59](https://user-images.githubusercontent.com/60083437/210896314-c6084afa-2ee1-4bbc-a8c1-973a3970ca1d.png)
***Your Design***
![Screenshot from 2023-01-06 04-59-29](https://user-images.githubusercontent.com/60083437/210896357-71032b0d-2cc2-445c-9bec-f5090a4a9ae1.png)